src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -98,6 +99,35 @@ def test_get_os_options(self):
                 'type', 'Microsoft.ContainerService/locations/osOptions')
         ])
 
+    @unittest.skipUnless(os.getenv('OPENAI_API_KEY'), 'Skipped as not running with OPENAI_API_KEY')
```

Comment:
Can you leave a note (or a link) about where/how to acquire these keys in the command help message?